amazon-qldb-driver-core/src/driver.rs

```diff
@@ -82,15 +96,12 @@ impl QldbDriverBuilder {
             .max_size(self.max_concurrent_transactions)
             .connection_timeout(Duration::from_secs(10))
             .error_sink(Box::new(QldbErrorLoggingErrorSink::new()))
-            .build(QldbSessionManager {
-                client: client.clone(),
-                ledger_name: ledger_name.clone(),
-            })
-            .await?;
+            .build(QldbSessionV2Manager::new(client, ledger_name.clone()))
+            .await
```

Comment:
The bb8 doc says that the future completes when the pool has the requested number of connections open. Since you're not specifying min_idle, we will have a pool with zero sessions initially, right?

Reply:
I can't remember now, but yeah I think so.

I was thinking about making APIs that mirror the bb8 pool (i.e. exposing min_idle on our builder), but I'm not sure. Got any thoughts?